ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -23,6 +23,10 @@ int frames_count = 0;
 unsigned long current_time_mcs = 0;
 bool game_window_focused = true;
 
+// located in actions.h but not included
+// to avoid multiple definitions
+std::string game_format_caption();
```

Comment:
@JoshDreamland `GAME_GLOBALS.h` is not separating declaration from definition and included from `SHELLmain.cpp` which is why I had to do this.